pkg/web_css/lib/src/_base.scss

```diff
@@ -41,17 +44,39 @@ body,
   font-weight: 400;
   font-size: 16px;
 
-  h1, h2, h3, h4, h5, h6 {
+  h1,
+  h2,
+  h3,
+  h4,
+  h5,
+  h6 {
     font-family: var(--pub-default-text-font_family);
     font-weight: 400;
   }
 
-  h1 { font-size: 36px; }
-  h2 { font-size: 24px; }
-  h3 { font-size: 18px; }
-  h4 { font-size: 18px; }
-  h5 { font-size: 16px; }
-  h6 { font-size: 16px; }
+  h1 {
```

Comment:
Can we revert these changes? Even if we want to have a new formatting, it would be better to have a separate PR for them.